cirq/devices/noise_model.py

```diff
@@ -70,21 +51,8 @@ def noisy_moments(self, moments: 'Iterable[cirq.Moment]',
             A sequence of OP_TREEs, with the k'th tree corresponding to the
             noisy operations for the k'th moment.
         """
+        return list(moments)
```

Comment:
Why was this change made? The purpose of the alternatives is that the user can implement any one of the methods and the rest will be derived for them. Now if the user implements one method the others will have no noise, which is a bug.

Reply:
The intention here is that `noisy_operation` and `noisy_moment` generate different noise behaviors. For example, the `noise_model_test.MarkovianPairwiseCorrelatedNoise` is applied only at the moment level, affecting each operation in a given moment. The default `noisy_operation` function returns input operations unchanged because the class defining only `noisy_moment` implied that the user didn't have operation-specific noise in mind.

The reasoning here is that to apply different noise models at different levels of the operation-moment-moments hierarchy they must exist within a single model. Otherwise, subsequent noise models might apply their noise to operations that were introduced to previous noise models. This is what I'm demonstrating in `noise_model_test.test_composition_of_noise_models`

Reply:
of course, this is just the structure I'm proposing. There might be better ways to deal with applying multiple noise models without applying noise operations on top of noise operations.